### DISCUSSION TITLE
Add a command /statstop [stats_option] just like /balancetop

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#7271] [Feature Requests]
Add a command /statstop [stats_option] just like /balancetop.
I mean:  /statstop block_break, /statstop kill_mobs, /statstop play_time etc.
It's very useful for some rank porpose

**Comment by mrfloris** [comment]
You could consider using a plugin like this: <https://www.spigotmc.org/resources/ajleaderboards.85548/> which you give the placeholder and then it will start tracking users. Generating a top x list for you.
